LICENSE.md

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2015-2018 Okta Inc.
+Copyright (c) 2015-2022 Okta Inc.
```

Comment:
This should be updated to 2015-Present per our license talks

Reply:
I will make it so!